README.md

```diff
@@ -2,38 +2,49 @@
 
 A Sourcegraph extension that provides code intelligence for TypeScript.
 
-## How to deploy the server
+## How to deploy the language server
```

Comment:
This was somewhat intentional - the "server" actually wraps the actual "language server".

Reply:
And the "language server" actually wraps "tsserver", which doesn't speak LSP at all. Since this is all implementation details, I intentionally called it just "server".